static/src/javascripts/projects/common/modules/commercial/reader-revenue-banner.js

```diff
@@ -14,11 +13,7 @@ let data: ?BannerDataResponse = null;
 const show = () => data ? renderBanner(data) : Promise.resolve(false);
 
 const canShow = (): Promise<boolean> => {
-    const countryCode = geolocationGetSync();
-    const forceBanner = window.location.search.includes('force-remote-banner=true');
```

Comment:
Aren't you losing the ability to force the banner here?

Reply:
I figured that `forceBanner` was introduced to make it easier to test because of the restriction to limit the banner to Australia in frontend, which I've now removed, a side affect of doing this is it should make it easier to test without needing to `forceBanner`.

Also, this `forceBanner` boolean was never honoured by the `contributions-service` that has always also checked whether the user is in `Australia`, so removing it shouldn't make it harder to test.

Reply:
It's worth noting that in order to use such a parameter, it would have to be registered in `DevParametersHttpRequestHandler` :

https://github.com/guardian/frontend/blob/26ce951a638900058d3e747f7ac84dfef5b43411/common/app/dev/DevParametersHttpRequestHandler.scala#L20-L53